.changeset/few-parrots-draw.md

```diff
@@ -0,0 +1,5 @@
+---
+'@cloudfour/patterns': minor
+---
+
+Imports Wordpress Gutenberg block-library CSS into Gutenberg stories
```

Comment:
I don't believe this change should bump the pattern library version, since it doesn't impact the CSS output for other projects to consume (only the Storybook docs).

Reply:
I have removed the changeset file from this PR.